docs/quickstarts/react.mdx

```diff
@@ -212,13 +209,15 @@ You're ready to [deploy your app to production](/docs/deployments/overview) and
 
 ## You're authenticated!
 
-Congratulations! Your app is now only available to authenticated users! But this is just the first step. If you would like to clone or investigate this application, please check out:
```

Comment:
So I feel like this sentence is a bit misleading. This set up is not like Nextjs where the middleware is protecting every single page ootb. In this tutorial, we have set up some content that can be seen by un-authenticated users, so technically, the application is *not* only available to authenticated users - there's still content that is served to un-authenticated users.

Reply:
The app behaviour was changed based on suggestions but I missed this.

Reply:
and do the rest of the changes look fair to you?

Reply:
For sure. I didn't have time earlier today.